source/_patterns/01-molecules/navigation/footer-navigation.twig

```diff
@@ -0,0 +1,13 @@
+{% if footer_nav %}
+<nav class="footer__nav">
+    <ul class="inline-list">
+        {% for item in footer_nav.items %}
+            <li class="footer__nav-item inline-list__item">
+                <a href="{{ item.url }}" class="footer__nav-link font--secondary link--white">
+                    {{ item.text }}
+                </a>
+            </li>
+        {% endfor %}
+    </ul>
+</nav> <!-- /.footer__nav -->
+{% endif %}
```

Comment:
_Formatting:_ Tabs look inconsistent in this twig file.

Reply:
You are right. I saw twigs with two spaces indent, and other with four. What is the main convention to use @tpitre ?

Reply:
Ok, I found my answer from here : https://github.com/adventistchurch/alps/pull/242/files :)